recipes/python-for-android/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "python-for-android" %}
+  {% set version = "2019.7.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 701bf784958a958d70222a6d530f5cf2b44603e183638b35fa2a3de6476afba4
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```

Comment:
Do you need the ``C`` compiler?  It seems that you are not compiling anything, am I right?

Reply:
I think you can delete this part